prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml

```diff
@@ -5,15 +5,48 @@ presubmits:
     always_run: false
     decorate: true
     spec:
+      hostNetwork: true
       containers:
-      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+      - image: quay.io/ansible/ansible-runner:stable-2.9-latest
         command:
-        - /bin/bash
-        args:
-        # probing the pod created
-        - "true && id && echo && hostname && echo && pwd && echo && ls -l && echo && whoami && echo && env && echo && df -h"          
+        # . represents the base directory of the cloned bms-toolkit repo, which is bms-toolkit
+        # the presubmit_tests dir and test_pr.sh are non-existent at this point, 
+        # but will be created in a test PR as part of testing the prowjob
+        - ./presubmit_tests/test_pr.sh        
         resources:
           requests:
             memory: "2.0Gi"
             cpu: "3.0"
-       
\ No newline at end of file
+        volumeMounts:
```

Comment:
These volumes/volumeMounts directives are kubectl-tested as in: (internal): http://b/202240337#comment22